translations/messages-en.properties

```diff
@@ -286,12 +286,14 @@ Replace\ country\ code\ for = Replace country code for
 Messaging\ window = Messaging window
 Send\ scheduled\ messages\ between = Send scheduled messages between
 and = and
-Upload\ Forms = Upload forms
```

Comment:
Deleting translation keys here doesn't work as you'd expect (they'll keep coming back). Read the [translation documentation](https://github.com/medic/medic-docs/blob/master/development/translations.md#modifying-or-removing-translation-keys).